src/nvapi/lfx.cpp

```diff
@@ -12,7 +12,7 @@ namespace dxvk {
         if (m_lfxModule != nullptr)
```

Comment:
Looks like you missed some `!= nullptr` checks.

Reply:
Thanks, addressed where it doesn't hurt readability, e.g. I left this notation when used in return statements.